### PR TITLE
kubesphere-3.2.0-ga-announcement

### PR DESCRIPTION
**Pull request by Bettygogo2021**
Signed-off-by: Bettygogo2021 <bintinggou@yunify.com>
Hi, pls help me review this doc. 3ks. #2051 
/assign @FeynmanZhou @JiannaLiu

### CODE REVIEW

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+### **GPU scheduling and quota management**
+
+With the rapid development of artificial intelligence (AI) and machine learning, more and more AI companies are calling for GPU resource scheduling and management features for server clusters, especially monitoring of GPU usage and management of GPU resource quotas. To address users' pain points, KubeSphere 3.2.0 makes our original GPU management even easier.
+
+KubeSphere 3.2.0 allows you to create GPU workloads on the GUI, schedule GPU resources, and manage GPU resource quotas by tenant. Specifically, it can be used for NVIDIA GPU and vGPU solutions.
+
+![00-GPU-scheduling-quota-manage](/images/blogs/en/release-announcement3.2.0/00-GPU-scheduling-quota-manage.png)
+
+### **Enhanced observability**
```
Comment (JiannaLiu) [inline content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:33]: Enhanced Kubernetes observability

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+![04-cluster-manage](/images/blogs/en/release-announcement3.2.0/04-cluster-manage.png)
+
+KubeSphere 3.2.0 performs better in cross-cluster scheduling. When you are creating a federated Deployment across clusters, you can directly specify the number of replicas scheduled to each cluster. In addition, you can also specify the total number of replicas and weight of each cluster, and allow the system to automatically schedule replicas to each cluster according its weight. This feature is pretty helpful when you want to flexibly scale your Deployment and proportionally distribute replicas to multiple clusters.
+
+![05-federated-deployment](/images/blogs/en/release-announcement3.2.0/05-federated-deployment.png)
+
+![06-view-federation](/images/blogs/en/release-announcement3.2.0/06-view-federation.png)
+
+### **Operations and maintenance-friendly storage management**
```
Comment (JiannaLiu) [inline content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:71]: Is 'Operations-and-maintenance-friendly' better in this case？ I'm not sure though.

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+### **App Store open to community partners**
+
+The App Store and application lifecycle management are unique features of KubeSphere, which are based on self-developed and open-source [OpenPitrix](https://github.com/openpitrix/openpitrix).
+
+KubeSphere 3.2.0 adds the feature of **dynamically loading community-developed Helm charts into the KubeSphere App Store.** You can send a pull request containing the Helm chart of a new app to the App Store chart repository. After the pull request is merged, the app is automatically loaded to the App Store regardless of the KubeSphere version. Welcome to submit your Helm charts to https://github.com/kubesphere/helm-charts. Nocalhost and Chaos Mesh have integrated their Helms charts into KubeSphere 3.2.0 by using this method, and you can easily install them to your Kubernetes clusters by one click.
+
+![12-app-store](/images/blogs/en/release-announcement3.2.0/12-app-store.png)
+
+### **More independent KubeSphere DevOps**
```
Comment (JiannaLiu) [inline content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:111]: KubeSphere DevOps -> Kubernetes DevOps (on KubeSphere）

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+KubeSphere DevOps has developed into an independent project ks-devops in KubeSphere v3.2.0, which is intended to allow users to run KubeSphere DevOps in any Kubernetes clusters. Currently, you can use a Helm chart to install the backend components of ks-devops.
+
+Jenkins is a CI engine with a large user base and a rich plug-in ecosystem. In KubeSphere 3.2.0, we will let Jenkins do what it is good at—functioning only as an engine in the backend to provide stable pipeline management capability. A newly added CRD PipelineRun encapsulates run records of pipelines, which reduces the number of APIs required for directly interacting with Jenkins and boosts performance of CI pipelines.
+
+Starting from KubeSphere v3.2.0, KubeSphere DevOps allows you to build images by using pipelines based on containerd. As an independent project, KubeSphere DevOps will support independent deployment of the backend and frontend, introduce GitOps tools such as Tekton and ArgoCD, as well as integrate project management and test management platforms.
+
+### **Flexible cluster deployment**
```
Comment (JiannaLiu) [inline content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:119]: Kubernetes cluster

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
+---
+title: 'Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways'
```
Comment (FeynmanZhou) [inline content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:2]: KubeSphere 3.2.0 GA: Brings AI-oriented GPU Scheduling and Flexible Gateway

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+![05-federated-deployment](/images/blogs/en/release-announcement3.2.0/05-federated-deployment.png)
+
+![06-view-federation](/images/blogs/en/release-announcement3.2.0/06-view-federation.png)
+
+### **Operations and maintenance-friendly storage management**
+
+Persistent storage is the most important focus for enterprises running Kubernetes in their production environment, and stable and reliable storage guarantees the safety of core enterprise data. On the KubeSphere 3.2.0 web console, the **Volumes** feature allows the administrator to decide whether to enable volume cloning, snapshot capturing, and volume expansion, making persistent storage operations and maintenance for stateful apps more convenient.
```
Comment (FeynmanZhou) [inline content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:73]: Is it necessary to keep `,and` between `environment` and `stable`? Shall we divide it into two individual sentences?

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+![07-volume-manage](/images/blogs/en/release-announcement3.2.0/07-volume-manage.png)
+
+The default immediate binding mode binds a volume to a backend storage device immediately when the volume is created. This mode does not apply to storage devices with topology limits and may cause Pod scheduling failures. KubeSphere 3.2.0 provides the delayed binding mode to address this issue, which guarantees that a volume (PVC) is bound to a volume instance (PV) only after the volume is mounted to a Pod. This feature ensures that resources are properly scheduled based on Pod resource requests.
```
Comment (FeynmanZhou) [inline content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:77]: volume instance => Persistent Volume

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+
+![08-storage-class-settings](/images/blogs/en/release-announcement3.2.0/08-storage-class-settings.png)
+
+In addition to volume management, KubeSphere 3.2.0 now also supports volume instance management, and you can view volume instance information, edit volume instances, and delete volume instances on the web console.
```
Comment (FeynmanZhou) [inline content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:81]: Same, volume instance => Persistent Volume

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+
+The administrator can directly manage and configure all project gateways on the cluster gateway settings page without having to go to each workspace. The Kubernetes ecosystem provides many ingress controllers that can be used as the gateway solution. In KubeSphere 3.2.0, the gateway backend is refactored, which allows you to use any ingress controllers that support v1\\ingress as the gateway solution.
+
+![11-gateway-settings2](/images/blogs/en/release-announcement3.2.0/11-gateway-settings2.png)
+
+### **Authentication and authorization**
+
+A unified and all-round identity management and authentication system is indispensable for logical isolation in a multi-tenant system. Apart from support for AD/LDAP and OAuth 2.0 identity authentication systems, KubeSphere 3.2.0 also provides a built-in authentication service based on OpenID Connect to provide authentication capability for other components. OpenID Connect is a simple user identity authentication protocol based on OAuth 2.0 with a bunch of features and security options to meet enterprise-grade business requirements.
+
+### **App Store open to community partners**
```
Comment (FeynmanZhou) [inline content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:103]: IMO, if we use `open` as a verb, we might use `opens`

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+
+The App Store and application lifecycle management are unique features of KubeSphere, which are based on self-developed and open-source [OpenPitrix](https://github.com/openpitrix/openpitrix).
+
+KubeSphere 3.2.0 adds the feature of **dynamically loading community-developed Helm charts into the KubeSphere App Store.** You can send a pull request containing the Helm chart of a new app to the App Store chart repository. After the pull request is merged, the app is automatically loaded to the App Store regardless of the KubeSphere version. Welcome to submit your Helm charts to https://github.com/kubesphere/helm-charts. Nocalhost and Chaos Mesh have integrated their Helms charts into KubeSphere 3.2.0 by using this method, and you can easily install them to your Kubernetes clusters by one click.
+
+![12-app-store](/images/blogs/en/release-announcement3.2.0/12-app-store.png)
+
+### **More independent KubeSphere DevOps**
+
+KubeSphere DevOps has developed into an independent project ks-devops in KubeSphere v3.2.0, which is intended to allow users to run KubeSphere DevOps in any Kubernetes clusters. Currently, you can use a Helm chart to install the backend components of ks-devops.
```
Comment (FeynmanZhou) [inline content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:113]: Please add a link: ks-devops => [ks-devops](https://github.com/kubesphere/ks-devops)

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+
+Starting from KubeSphere v3.2.0, KubeSphere DevOps allows you to build images by using pipelines based on containerd. As an independent project, KubeSphere DevOps will support independent deployment of the backend and frontend, introduce GitOps tools such as Tekton and ArgoCD, as well as integrate project management and test management platforms.
+
+### **Flexible cluster deployment**
+
+If you do not have a Kubernetes cluster, you can use KubeKey to install both Kubernetes and KubeSphere; if you already have a Kubernetes cluster, you can use ks-installer to install KubeSphere only.
+
+[KubeKey](https://github.com/kubesphere/kubekey) is an efficient open-source cluster deployment tool in the KubeSphere community, which uses Docker as the default container runtime. It can also use CRI runtimes such as containerd, CRI-O, and iSula. You can use KubeKey to deploy an etcd cluster independent of Kubernetes for better flexibility.
```
Comment (FeynmanZhou) [inline content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:123]: [KubeKey](https://github.com/kubesphere/kubekey) is an efficient open-source installer.

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md (hunk 1 above)
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:33]: Fixed.

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+
+***Observability is the ability to measure the internal states of a system by examining its outputs.*** A system is considered "observable" if the current state can be estimated by only using information from outputs, namely telemetry data collected by the three pillars of observability: logging, tracing and metrics.
+
+1. More powerful custom monitoring dashboards
+
+KubeSphere 3.1.0 has added the cluster-level custom monitoring feature, which allows you to generate custom monitoring dashboards by selecting a default template, uploading a template, or customizing a template. KubeSphere 3.2.0 provides a default template for creating a Grafana monitoring dashboard. You can import a Grafana monitoring dashboard by specifying the URL or uploading the JSON file of the dashboard, and then KubeSphere will automatically convert the Grafana monitoring dashboard into a KubeSphere cluster dashboard.
```
[the remark this answers is not in the document]
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:41]: Fixed.

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+
+- In addition to the various notification channels such as email, DingTalk, WeCom, webhook, and Slack, KubeSphere 3.2.0 now also allows you to test and validate the notification channels you configure.
+
+![03-platform-settings](/images/blogs/en/release-announcement3.2.0/03-platform-settings.png)
+
+3. On the etcd monitoring page, the system automatically adds the `Leader` tag to the etcd leader.
+
+### **Multi-cloud and multi-cluster management**
+
+CNCF Survey 2020 shows that over 80% of users run more than two Kubernetes clusters in their production environment. KubeSphere aims at addressing multi-cluster and multi-cloud challenges. It provides a unified control plane and supports distributing applications and replicas to multiple clusters deployed across public cloud and on-premises environments. Moreover, KubeSphere supports observability across clusters, including features such as monitoring, logs, events, and auditing logs of multiple dimensions.
```
[the remark this answers is not in the document]
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:61]: fixed.

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md (hunk 2 above)
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:71]: Fixed

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md (hunk 3 above)
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:111]: fixed

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md (hunk 4 above)
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:119]: fixed

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 14]
```diff
@@ -0,0 +1,147 @@
+---
+title: 'Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways'
+tag: 'KubeSphere, release'
+keyword: 'Community, Open-source, Contribution, KubeSphere, release, AI, GPU'
```
[the remark this answers is not in the document]
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:4]: fixed

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 15]
```diff
@@ -0,0 +1,147 @@
...
+description: 'KubeSphere 3.2.0 supports GPU resource scheduling and management and GPU usage monitoring, which further improves user experience in cloud-native AI scenarios. Moreover, enhanced features such as Multi-cluster management, Multi-tenant management,Observability, DevOps, App Store, and Service governance further perfect the interactive design for better user experience.'
```
[the remark this answers is not in the document]
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:5]: fixed

[the remark this answers is not in the document]
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:5]: fixed

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 16]
```diff
@@ -0,0 +1,147 @@
...
+createTime: '2021-11-03'
+author: 'KubeSphere'
+snapshot: '/images/blogs/en/release-announcement3.2.0/v3.2.0-GA-cover.png'
+---
+
+![3.2.0GA](/images/blogs/en/release-announcement3.2.0/3.2.0GA.png)
+
+No one would ever doubt that **Cloud Native** has become the most popular service technology. KubeSphere, a distributed operating system for cloud-native application management with Kubernetes as its kernel, is definitely one of the tide riders surfing the cloud-native currents. KubeSphere has always been upholding the commitment of 100% open source. Owing to support from the open-source community, KubeSphere has rapidly established a worldwide presence.
+
+On November 2, 2021, we are excited to announce this news: KubeSphere 3.2.0 is officially released!
```
[the remark this answers is not in the document]
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:15]: fixed

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 17]
```diff
@@ -0,0 +1,147 @@
...
+
+Do you still remember KubeSphere 3.1.0 released six months ago? It brings you amazing features, such as **Edge Computing** and **Metering and Billing**, which extend Kubernetes from the cloud to the edge to help enterprises more clearly assess the operating cost of infrastructure and applications. Do you still remember KubeSphere 3.1.1 released three months ago? It allows you to specify an existing Prometheus instance in the Kubernetes cluster while deploying KubeSphere, and the more interactive design improves user experience.
+
+Today, feature-rich KubeSphere 3.2.0 won't let you down. **GPU resource scheduling and management** and GPU usage monitoring further improve user experience in cloud-native AI scenarios. Moreover, enhanced features such as **Multi-cluster management**, **Multi-tenant management,** **Observability**, **DevOps**, **App Store**, and **Service governance** further perfect the interactive design for better user experience.
```
[the remark this answers is not in the document]
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:19]: Fixed, and deleted column 17.

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md [hunk 18]
```diff
@@ -0,0 +1,147 @@
...
+---
+
+![3.2.0GA](/images/blogs/en/release-announcement3.2.0/3.2.0GA.png)
+
+No one would ever doubt that **Cloud Native** has become the most popular service technology. KubeSphere, a distributed operating system for cloud-native application management with Kubernetes as its kernel, is definitely one of the tide riders surfing the cloud-native currents. KubeSphere has always been upholding the commitment of 100% open source. Owing to support from the open-source community, KubeSphere has rapidly established a worldwide presence.
+
+On November 2, 2021, we are excited to announce this news: KubeSphere 3.2.0 is officially released!
+
+Do you still remember KubeSphere 3.1.0 released six months ago? It brings you amazing features, such as **Edge Computing** and **Metering and Billing**, which extend Kubernetes from the cloud to the edge to help enterprises more clearly assess the operating cost of infrastructure and applications. Do you still remember KubeSphere 3.1.1 released three months ago? It allows you to specify an existing Prometheus instance in the Kubernetes cluster while deploying KubeSphere, and the more interactive design improves user experience.
```
[the remark this answers is not in the document]
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:17]: Deleted

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:41]: Fixed

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:61]: fixed

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md (hunk 6 above)
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:73]: fixed

File: content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md (hunk 9 above)
Comment (Bettygogo2021) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:103]: 3KS for your advice. But here "open" is used as an adjective

Comment (FeynmanZhou) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:103]: Will `App Store is open to community partners` looks much better?

Comment (JiannaLiu) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:103]: I think 'is' can be omitted here

Comment (FeynmanZhou) [reply content/en/blogs/Brand-new KubeSphere 3.2.0 Brings AI-oriented GPU Scheduling and Ever More Flexible Gateways.md:103]: Okay.

**Pull request closed without merging**